### PR TITLE
Add release notes for 0.38.0

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by Callisto13 — APPROVED** [review]

File: docs/release_notes/0.38.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- Fix creating private cluster in some regions (#3228)
+- Fix suspend ASG processes for nodegroups (#3218)
+- Support position name in get iamserviceaccount (#3205)
+- Support parsing YAML Cloudformation templates (#3170)
+- Refactor iamserviceaccounts  (#3135)
+- Do not error for empty get nodegroups if output is yaml or json (#3178)
+
+## Acknowledgments
+Weaveworks would like to sincerely thank:
+ @Legion2, @dbluxo, @dholbach and @tkms0106
```
Comment (Callisto13) [inline docs/release_notes/0.38.0.md:33]: oh wait should dholbach be here? isn't he one of ours?

Comment (michaelbeaumont) [reply docs/release_notes/0.38.0.md:33]: :shrug: he's not on the ignore list

Comment (Callisto13) [reply docs/release_notes/0.38.0.md]: 🤷‍♀️

File: docs/release_notes/0.38.0.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+- Support sourceSecurityGroupIds for unmanaged nodegroups (#3254)
+- added RoleOnly option for iamserviceaccounts (#3122)
+- Flux v2 support (#3066)
+- Add support for well known policies with IRSA (#3045) 
+
+## Improvements
+
+- Add option to pass kubeconfig path to Flux 2 installer (#3256)
+- always enable log timestamps (#3251)
```
Comment (Callisto13) [inline docs/release_notes/0.38.0.md:13]:
```suggestion
- Always enable log timestamps (#3251)
```

File: docs/release_notes/0.38.0.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Release 0.38.0
+
+## Features
+
+- Support sourceSecurityGroupIds for unmanaged nodegroups (#3254)
+- added RoleOnly option for iamserviceaccounts (#3122)
```
Comment (Callisto13) [inline docs/release_notes/0.38.0.md:6]:
```suggestion
- Add RoleOnly option for iamserviceaccounts (#3122)
```

File: docs/release_notes/0.38.0.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- Flux v2 support (#3066)
+- Add support for well known policies with IRSA (#3045) 
+
+## Improvements
+
+- Add option to pass kubeconfig path to Flux 2 installer (#3256)
+- always enable log timestamps (#3251)
+- Add deprecation notice to enable repo command (#3206)
+- Detect and log unsupported region error for fully-private clusters (#3186)
+- ignore ResourceNotFoundException error during cluster deletion (#3210)
```
Comment (Callisto13) [inline docs/release_notes/0.38.0.md:16]:
```suggestion
- Ignore ResourceNotFoundException error during cluster deletion (#3210)
```

**Pull request merged**